middleware.go

```diff
@@ -21,8 +26,70 @@ type AuthlessHealthServer struct {
 	IgnoreAuthMixin
 }
 
-// NewAuthlessServer returns a new gRPC health server that ignores auth
+// NewAuthlessHealthServer returns a new gRPC health server that ignores auth
 // middleware.
 func NewAuthlessHealthServer() *AuthlessHealthServer {
 	return &AuthlessHealthServer{Server: health.NewServer()}
 }
+
+// SetServicesHealthy sets the service to SERVING
+func (s *AuthlessHealthServer) SetServicesHealthy(svcDesc ...*grpc.ServiceDesc) {
+	for _, d := range svcDesc {
+		s.SetServingStatus(
+			d.ServiceName,
+			healthpb.HealthCheckResponse_SERVING,
+		)
+	}
+}
+
+// DefaultUnaryMiddleware is a recommended set of middleware that should each gracefully no-op if the middleware is not applicable.
```

Comment:
This doesn't look like 80char lines.
I need to fix CI to actually lint things correctly.
I can fix it in a follow up.

Reply:
fixed - also added a test